Xamarin.Android.sln

```diff
@@ -85,6 +87,8 @@ Global
 		Release|AnyCPU = Release|AnyCPU
 		XAIntegrationDebug|AnyCPU = XAIntegrationDebug|AnyCPU
 		XAIntegrationRelease|AnyCPU = XAIntegrationRelease|AnyCPU
+		XAIntegrationDebug|Any CPU = XAIntegrationDebug|Any CPU

```

Comment:
This and the following line almost *certainly* breaks `msbuild` builds. See also PR #74:

> There were a few issues that msbuild didn't like:
> 
> 1. MSBuild doesn't like the "extra" configuration mappings in Xamarin.Android.sln.

This is *exactly* what is being referred to here.